app/v2/connectors/BaseConnector.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.connectors
+
+import connectors.util.CustomHttpReader
+import connectors.util.CustomHttpReader.INTERNAL_SERVER_ERROR
+import play.api.http.HeaderNames
+import play.api.http.MimeTypes
+import play.api.libs.json.Reads
+import play.api.mvc.RequestHeader
+import uk.gov.hmrc.http.Authorization
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.http.HttpErrorFunctions
+import uk.gov.hmrc.http.HttpResponse
+
+class BaseConnector extends HttpErrorFunctions {
+
+  protected val requestHeaders: Seq[(String, String)] =
+    Seq((HeaderNames.CONTENT_TYPE, MimeTypes.XML))
+
+  def responseHeaders(): Seq[(String, String)] =
+    Seq((HeaderNames.CONTENT_TYPE, MimeTypes.JSON))
+
+  protected val mdtpString = "MDTP-%s-%d-%d"
```

Comment:
This string format isn't used in Phase 5, the correlation ID is simply `movementId-messageId`

Reply:
@drnaylor-oc  this format string is used for populating the header: "X-Message-Recipient".
So my assumption is you mean set the header to movementId-messageId?

Reply:
Well, we don't actually have it at all, and we don't use the `X-Message-Recipient` header in P5. In P5, the correlation ID is part of the header, and you're right, we don't have the message ID. We just want to set it to 16 zeroes in the TS API.